src/App.tsx

```diff
@@ -42,54 +63,101 @@ export function getReorderedGoods(
 }
 
 // DON'T save goods to the state
-// type State = {
-//   isReversed: boolean,
-//   sortType: SortType,
-// };
-
-export const App: React.FC = () => {
-  return (
-    <div className="section content">
-      <div className="buttons">
-        <button
-          type="button"
-          className="button is-info is-light"
-        >
-          Sort alphabetically
-        </button>
-
-        <button
-          type="button"
-          className="button is-success is-light"
-        >
-          Sort by length
-        </button>
-
-        <button
-          type="button"
-          className="button is-warning is-light"
-        >
-          Reverse
-        </button>
-
-        <button
-          type="button"
-          className="button is-danger is-light"
-        >
-          Reset
-        </button>
-      </div>
+type State = {
+  isReversed: boolean,
+  sortType: SortType,
+  resetBTN: boolean,
+};
+
+export class App extends Component<{}, State> {
+  state: State = {
+    isReversed: false,
+    sortType: SortType.NONE,
+    resetBTN: false,
+  };
```

Comment:
Instead of using resetBTN, you can create a method that resets state to default values
```suggestion
  state: State = {
    isReversed: false,
    sortType: SortType.NONE,
  };
```